lib/real-debrid.js

```diff
@@ -100,18 +125,25 @@ async function listTorrents(apiKey, skip = 0) {
     return metas || []
 }
 
-async function listTorrentsParallel(apiKey, page = 1, pageSize = 50) {
+async function listFilesParrallel(fileType, apiKey, page = 1, pageSize = 50) {
     const RD = new RealDebridClient(apiKey, {
         params: {
             page: page,
             limit: pageSize
         }
     })
 
-    const torrents = await RD.torrents.get(0, page, pageSize)
-        .catch(err => handleError(err))
-
-    return torrents || []
+    let files = []
+    if (fileType == FILE_TYPES.TORRENTS) {
+        files = await RD.torrents.get(0, page, pageSize)
+           .catch(err => handleError(err))
+    } else if (fileType == FILE_TYPES.DOWNLOADS) {
+        files = await RD.downloads.get(0, page, pageSize)
```

Comment:
Fixed - the previous RD API code wasn't using the parameters properly so it would always return the same 50 results despite asking for different pages.
I had to fix it here: https://github.com/atombarel/node-real-debrid/pull/4